repo/fs/fs.go

```diff
@@ -89,9 +78,10 @@ func NewRepo(path string, fsys *muxfs.Mux, book *logbook.Book, cache *dscache.Ds
 		return nil, err
 	}
 
+	own := pro.Owner()
 	// add our own profile to the store if it doesn't already exist.
```

Comment:
I feel like a better approach would be to have the interface guarantee that the owner is always in the profile list. Would probably simplify understanding how to use it. This is fine for now, but it definitely feels like more a work-around than a semantic we actually want to support.